File: backend/src/main/java/com/kiwi/features/personality/data/PersonalityDTO.java
package com.kiwi.features.personality.data;

import lombok.*;

import java.util.List;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder @EqualsAndHashCode @ToString
public class PersonalityDTO {
    private String realName;
    private String knightName;
    private String build;
    private List<String> goodApps;
    private List<String> badApps;
    private List<String> neutralApps;
}

File: mobile/app/src/main/java/com/bellako/kiwi/features/personality/data/PersonalityDTO.kt
package com.bellako.kiwi.features.personality.data

data class PersonalityDTO(
    val realName: String,
    val knightName: String,
    val build: String,
    val goodApps: List<String>,
    val badApps: List<String>,
    val neutralApps: List<String>,
)
